src/js/controller/controller.js

```diff
@@ -511,6 +511,10 @@ Object.assign(Controller.prototype, {
                     });
                 }
 
+                if (!_this.getMute()) {
```

Comment:
You can shift this conditional into the above block to avoid calling `getMute` twice